src/utils/patches.js

```diff
@@ -387,3 +392,46 @@ export const addPrefixToPatch = (prefix, patch) => ({
   ...patch,
   path: `${prefix}${patch.path}`,
 });
+
+const getDeviceType = device => {
+  let type;
+  if (isNic(device)) {
```

Comment:
again, we should already know here what kind of device we have

Reply:
I guess easiest way is to wrap the disks/nic in a object with type attribute.

Reply:
Done!